# Mmirus

## Github repository has code for the different analyses conducted

## 1. Phylogenetics

## 1.1 ddRAD phylogeny
1.1.1 stacks pipeline: details how multi-species data were analysed using stacks v2.41

1.1.2 ddRAD R QC code: details how R code was used to QC SNP data from stacks pipeline, select loci and samples, and generate dataset for IQtree

1.1.3. SNP IQtree code: details how IQtree was run to generate SNP based phylogeny

## 1.2 mitochondrial phylogeny
1.2.1 mitogenome pipeline: details how the phylogenetic tree was generated with BEAST2.

1.2.2 node calibrations: details of calibrations implemented in BEAUTI when preparing the XML file for BEAST2

1.2.3 partitions.csv: details of PartitionFinder2 results and partitions imported into BEAUTI when preparing the XML file for BEAST2

1.2.4 BEAST2.xml: details of BEAST2 input

1.2.5 Beast2.log.zip: combined log output of three seperate BEAST2 runs 

1.2.6 Beast2.trees.zip: combined tree file output from three seperate BEAST2 runs



## 2. Genomics analyses

## 2.1 Genome mapping and analyses
2.1.1 Raw read quality control and genome mapping: steps taken to trim filter and map reads

2.1.2 BAM downsampling: steps taken to downsample bam files

2.1.3 psmc pipeline: details how psmc files were generated and visualized

2.1.4 hybrid psmc (hpsmc): details how hpsmc files were generated and visualized

2.1.5 Heterozygosity calculation: details how genome-wide global heterozygosity was calculated in ROHan 

2.1.6 IBS analysis: details how the haploid identity-by-state anaylsis was conducted



